src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs

```diff
@@ -941,7 +947,7 @@ private List<(string key, string feed)> GetManagedPackageSources(HashSet<string>
 
             foreach (string feed in feeds)
             {
-                var parsedFeed = ParseMaestroManagedFeed(feed);
+                var parsedFeed = ParseFeed(feed);
```

Comment:
These will still be managed by Maestro but will be blob feeds instead of artifact feeds

Reply:
Reverted the name back